docker/utils/utils.py

```diff
@@ -615,6 +615,16 @@ def create_container_config(
         if volumes_from is not None:
             raise errors.InvalidVersion(message.format('volumes_from'))
 
+    # NetworkMode must be present and valid in host config from 1.20 onwards
+    if compare_version('1.20', version) >= 0:
+        if host_config is None:
+            host_config = {'NetworkMode': 'none'}
```

Comment:
Just looked at v1.6.0 docker\docker runconfig\parse.go, func parseNetMode L445. (Sorry, can't work out how to get a link directly to a previous version). The valid cases are bridge, none, host and container. Looks like default didn't arrive until later.